src/cli/onefuzz/job_templates/job_monitor.py

```diff
@@ -78,11 +78,21 @@ def has_files(self) -> Tuple[bool, str, Any]:
             None,
         )
 
+    def is_stopping(self) -> Tuple[bool, str, Any]:
+        tasks = self.onefuzz.tasks.list(job_id=self.job.job_id)
+        stopping = [
+            "%s:%s" % (x.config.task.type.name, x.state.name)
+            for x in tasks
+            if x.state not in TaskState.shutting_down()
+        ]
+        return (not stopping, "waiting on: %s" % ", ".join(sorted(stopping)), None)
+
     def wait(
         self,
         *,
         wait_for_running: Optional[bool] = False,
         wait_for_files: Optional[Dict[Container, int]] = None,
+        wait_for_stopping: Optional[bool] = False,
```

Comment:
This is a useful feature that I imagine we'll want to expose in the managed templates as well.  However, I think this is fine to leave for a follow-on PR